src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs

```diff
@@ -36,10 +38,126 @@
+                      {:header-component (header "Inputs") :starting-width 200
+                       :cell-renderer (fn [row-num conf] (cell (stringify_map (conf "inputs"))))}
+                      {:header-component (header "Outputs") :starting-width 200
+                       :cell-renderer (fn [row-num conf] (cell (stringify_map (conf "outputs"))))}
+                      {:header-component (header "Prerequisites") :starting-width 200
+                       :cell-renderer (fn [row-num conf] (cell (stringify_map (conf "prerequisites"))))}]
+            :data (:method-confs props)
+            :row-props (fn [row-num conf]
+                         {:style {:fontSize "80%" :fontWeight 500
+                                  :paddingTop 10 :paddingBottom 7
+                                  :backgroundColor (if (even? row-num) (:background-gray style/colors) "#fff")}})})])])})
+
+
+(defn- modal-import-background [state]
+  {:backgroundColor "rgba(82, 129, 197, 0.4)"
+   :display (when-not (:import-overlay-shown? @state) "none")
+   :overflowX "hidden" :overflowY "scroll"
+   :position "fixed" :zIndex 9999
+   :top 0 :right 0 :bottom 0 :left 0})
+
+(def ^:private modal-import-content
+  {
+   ;:transform "translate(-50%, 0px)" ORIGINAL
+   :transform "translate(-50%,0px)"
+   :backgroundColor (:background-gray style/colors)
+   :position "relative" :marginBottom 60
+   :top 60
+   ;:left "50%"  ORIGINAL
+   :left "50%"
+   ;:width 500 ORIGINAL
+   :width "75%"
+   })
+
+
+(defn- render-import-overlay [state refs]
+  (let [clear-import-overlay (fn []
+      (common/clear! refs "wsName" "wsDesc" "shareWith")
+      (swap! state assoc :import-overlay-shown? false))]
+    [:div {
+      :style (modal-import-background state)
+      :onKeyDown (common/create-key-handler [:esc] clear-import-overlay)}
+    [:div {:style modal-import-content}
+      [:div {:style {:backgroundColor "#fff"
+                     :borderBottom (str "1px solid " (:line-gray style/colors))
+                     :padding "20px 48px 18px"
+                     }}
+      [:div {}
+        [:div {:style {:paddingLeft "100%"    :halign "right"  :align "right"}} " "
+        [comps/Button {:icon :x
+                       :onClick #(swap! state assoc :import-overlay-shown? false)}]]
+        [:div {:style {:fontSize 24   :align "center" :textAlign "center"}} "Select Method Configurations For Import"]
+        [ImportWorkspaceMethodsConfigurationsList {:method-confs (utils/parse-json-string
+                                                                  (utils/->json-string
+                                                                    (create-mock-methodconfs)))}]]
+        [comps/Button {:style :add   :text "Add selected to workspace" :onClick
+                        ;TODO
+                        ; 1) acquire list of selected confs and
+                        ; 2) use orch API (post) to copy them to the workspace
+                       (fn [] nil)}]]]]))
```

Comment:
okay....my original thinking was for it to do nothing as an additional reminder that there's something to do for it.  This is a small change to the UI code.